streaming/src/test/java/org/apache/spark/streaming/LocalJavaStreamingContext.java

```diff
@@ -28,6 +29,7 @@
 
     @Before
     public void setUp() {
+        SparkContext$.MODULE$.stopActiveContext();
```

Comment:
Intellij and SBT were both complaining. So I did this.

I'll try to rebuild and see what happens.